bindings_ffi/src/mls.rs

```diff
@@ -71,6 +71,7 @@ pub async fn create_client(
     nonce: u64,
     legacy_signed_private_key_proto: Option<Vec<u8>>,
 ) -> Result<Arc<FfiXmtpClient>, GenericError> {
+    init_logger(logger);
```

Comment:
If you add this here. Can you remove it from line 133 https://github.com/xmtp/libxmtp/pull/846/files#diff-3a24c3e76565487a710ac9863ac05160128f4f90892e07849b555a6de43a6e8fR133

The double init causes errors in android for some reason and error logs will no longer show up in the android.

Reply:
removed here 0c68a692a8fd82bc2b5f8f69df2c62d4b833ea88